### DISCUSSION TITLE
JWT Lib in microdot-session

### DISCUSSION THREAD

**Discussion by mc0110** [miguelgrinberg/microdot#112] [Q&A]
Hi Miguel,

thanks for your great work. I try to use your framework on micropython ports. But unfortunately I can't load the additional libs (e.g. JWT). Which versions did you use for micropython?

Thank you in advance
Magnus

**Comment by miguelgrinberg** [comment]
The links to the dependencies are in the documentation. Are you following those instructions?
https://microdot.readthedocs.io/en/latest/extensions.html#maintaing-secure-user-sessions

**Comment by mc0110** [comment]
Great!! Thank a lot for the hint!

Super project!!
